apps/backend/src/datasources/stfc/UOWSClient.ts

```diff
@@ -0,0 +1,14 @@
+import dotenv from 'dotenv';
+
+import { UOWS } from '../../generated';
+
+dotenv.config({ path: process.env.CONFIG_PATH });
+
+export function createUOWSClient() {
+  return new UOWS({
+    BASE: process.env.EXTERNAL_UOWS_API_URL,
+    HEADERS: {
+      Authorization: `Api-key ${process.env.API_KEY}`,
```

Comment:
`API_KEY` looks like it's a new setting, but it looks like there's still a reference to the old `EXTERNAL_AUTH_TOKEN` one in StfcUserDataSource, and I can't find a PR in our config repo to add the new value.